#! /bin/sh /usr/share/dpatch/dpatch-run
## 90_manpage_update.dpatch by Daniel Leidert (dale) <daniel.leidert@wgdd.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: This patch simply fixes the manpage-has-useless-whatis-entry lintian
## DP:   warning. It was copied from upstreams CVS 1.0 branch.

@DPATCH@
diff -urNad trunk~/man/bluefish.1 trunk/man/bluefish.1
--- trunk~/man/bluefish.1	2005-09-21 13:37:21.000000000 +0200
+++ trunk/man/bluefish.1	2007-08-20 23:59:35.000000000 +0200
@@ -1,29 +1,38 @@
-.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.29.
 .TH BLUEFISH "1" "December 2004" "Bluefish HTML editor" "User Commands"
-.SH NAME
-bluefish \- manual page for bluefish HTML editor
-.SH SYNOPSIS
+.SH "NAME"
+bluefish \- a powerful WYWIWYG HTML editor
+.SH "SYNOPSIS"
 .B bluefish
 [\fIoptions\fR] [\fIfilename\fR]
-.SH DESCRIPTION
-bluefish HTML editor
-.PP
+.SH "DESCRIPTION"
+.B bluefish
+is a powerful WYWIWYG (What You Write Is What You Get) editor for
+experienced web designers and programmers based on the GTK2 GUI interface.
+It supports many programming and markup languages, but focuses on editing
+dynamic and interactive websites.
+.SH "OPTIONS"
 Currently accepted options are:
 .TP
-\fB\-s\fR           skip root check
+\fB\-s\fR
+Skip root check.
 .TP
-\fB\-v\fR           current version
+\fB\-v\fR
+Show current version.
 .TP
-\fB\-n\fR           open new window
+\fB\-n\fR
+Open (file in) a new window.
 .TP
-\fB\-p filename\fR  open project
+\fB\-p \fIfilename\fR
+Open the given project.
 .TP
-\fB\-h\fR           this help screen
-.PP
-Usage: ./bluefish [options] [filename]
-.SH AUTHOR
-This manual page was written for the Debian GNU/Linux distribution
-because the original program does not have a manual page.
+\fB\-h\fR
+Show help screen.
+.SH "SEE ALSO"
+The programm is fully documented at
+\fI\%http://bluefish.openoffice.nl/manual/\fR.
+.SH "AUTHOR"
+This manual page was written by \fBDavide Puricelli\fR <\&evo@debian.org\&>
+for the Debian GNU/Linux system (but may be used by others) because the
+original program does not have a manual page.
 .PP
-This manual page was written by Davide Puricelli <evo@debian.org>,
-for the Debian GNU/Linux system (but may be used by others).
+The manpage later was adopted and edited by the Bluefish Development Team.
